gateway_code/static/iot-lab-cn.cfg

```diff
@@ -1,14 +1,2 @@
-adapter_khz 1000
-
-# comstick ftdi device
-interface ft2232
-ft2232_layout "usbjtag"
-ft2232_device_desc "ControlNode"
-ft2232_vid_pid 0x0403 0x6011
-
-jtag_nsrst_delay 100
-jtag_ntrst_delay 100
-
-# use combined on interfaces or targets that can't set TRST/SRST separately
-reset_config trst_and_srst
-
+ftdi_device_desc "ControlNode"
+ftdi_vid_pid 0x0403 0x6011
```

Comment:
Maybe comment here, that it must overwrite the default value given in `openocd` official `iotlab-usb.cfg`

Reply:
The low level reason, it is that the M3 and A8-M3 have a 2 ports `FT2232 ` where the control node has a 4 port `FT4232`.